File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonBooks.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;
import java.util.List;

/**
 * Created by zawzaw on 27/12/17.
 */

public class GsonBooks
{
    @SerializedName("total")
    private int total;

    @SerializedName("data")
    private List<GsonBook> books;

    public int getTotal() {
        return total;
    }

    public List<GsonBook> getBooks() {
        return books;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonBooks.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 02/01/18.
 */

class GsonBooks {

    @SerializedName("total")
    val total: Int = 0

    @SerializedName("data")
    val books: List<GsonBook>? = null

}
